apps/native/app/src/utils/get-main-root.ts

```diff
@@ -10,41 +10,61 @@ import {
 import { getThemeWithPreferences } from './get-theme-with-preferences'
 import { testIDs } from './test-ids'
 
+type Screen = 'Inbox' | 'Wallet' | 'Home' | 'Applications' | 'More'
+
 type RightButtonProps = {
   unseenCount?: number
   theme?: ReturnType<typeof getThemeWithPreferences>
+  screen?: Screen
 }
 
 export const getRightButtons = ({
   unseenCount = notificationsStore.getState().unseenCount,
   theme = getThemeWithPreferences(preferencesStore.getState()),
+  screen,
```

Comment:
I think it would be more clear to the scope of the function to have rather icons instead of screens to choose from since the screens are all calling this function. Meaning screen can tell the function what kind of icon it wants without to provide it with its name.  aka notifications, settings, licesnseScan
And to be future proof to future screens it would be nice to allow array of icons, i.e. `['notifications', 'settings']` etc.
If there would be more than one icon per screen